<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterWidget</class>
 <widget class="QWidget" name="CharacterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="PB_CREATE">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="PB_DELETE">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="PB_SELECT">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="ClickableLabel" name="L_SLOT0">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="L_SLOT1">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="L_SLOT2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="L_SLOT3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pseudonym (4~14 characters) :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QPushButton" name="PB_CREATEFINAL">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="PB_CANCELCREATION">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="LE_PSEUDONYM"/>
       </item>
       <item row="0" column="0" rowspan="2" colspan="4">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Class</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="ClickableLabel" name="L_ADV">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/ladvmalems.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="ClickableLabel" name="L_MA">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/lmamalem.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Gender</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="ClickableLabel" name="L_MALE">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/lmales.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="ClickableLabel" name="L_FEMALE">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/lfemale.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Hairstyle</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="ClickableLabel" name="L_HAIRSTYLEA">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/lhairstyleas.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="ClickableLabel" name="L_HAIRSTYLEB">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/lhairstyleb.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Hair color</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="ClickableLabel" name="L_0">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l0s.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="ClickableLabel" name="L_1">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l1.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="ClickableLabel" name="L_2">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l2.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="ClickableLabel" name="L_3">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l3.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="ClickableLabel" name="L_4">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l4.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="ClickableLabel" name="L_5">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l5.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="ClickableLabel" name="L_6">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l6.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="ClickableLabel" name="L_7">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l7.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="ClickableLabel" name="L_8">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l8.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="ClickableLabel" name="L_9">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../guires.qrc">:/charcreation/l9.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>modules/clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../guires.qrc"/>
 </resources>
 <connections/>
</ui>
